typescript/infra/src/tx/govern-transaction-reader.ts

```diff
@@ -255,16 +327,24 @@ export class GovernTransactionReader {
       insight = `Unenroll remote routers for ${insights.join(', ')}`;
     }
 
+    if (
+      decoded.functionFragment.name ===
```

Comment:
feels a little weird to have this explicitly handled when we also have the general purpose ownable logic

could this just call into the general purpose logic? as we always know that a warp route is ownable